app/javascript/components/admin/AuditModal.js

```diff
@@ -257,6 +258,25 @@ class AuditModal extends React.Component {
             <b>User Signed In</b>
           </span>
         );
+      case 'manual_lock_reason':
+        // Generic audit message in case before & after values not provided
+        if (!change.details || !Array.isArray(change.details) || !change.details.length) {
+          return (
+            <span>
+              <b>Manually set Status</b>
+              <InfoTooltip tooltipTextKey={'manualLockReasonAudit'} location="right"></InfoTooltip>: Updated
```

Comment:
since this text is the only thing that's different, can we do a ternary operator to just determine the text?

Reply:
```
{!change.details || !Array.isArray(change.details) || !change.details.length ? ': Updated' : ': Changed from &quot;{change.details[0]}&quot; to &quot;{change.details[1]}&quot;'}
```

Reply:
Good suggestion, I will also make that revision for the other audits as well to keep them consistent.

Reply:
Addressed: [2a0e541](https://github.com/SaraAlert/SaraAlert/pull/1222/commits/2a0e541e181900b4c5d4bdb904fffd1a53ef229b)